### ISSUE TITLE
Fails on my own videos.

### ISSUE THREAD

**Issue by joshualeung** [hx173149/C3D-tensorflow#66]
Hey guys!
I followed the instructions and trained on the UCF101 dataset and it reports test accuracy of 86%+ with a batch size of 10 clips and 16 frames per clip. but  when I try it with my own video, almost none is predicted correctly. Is there something wrong or it's just overfitted?

**Comment by Nortinwell** [comment]
I have the same problem as you. Have you solved it yet？

**Comment by orrrrz** [comment]
nope. I think it's over fitted.

**Comment by babdulhakim2** [comment]
@joshualeung Please how did you train with you own videos, I'm having challenge figuring out how to fine tune the model to train on my own videos
